tests/file_uploads/tests.py

```diff
@@ -38,6 +38,7 @@
     '../..\\hax0rd.txt',        # Relative path, mixed.
     '..&#x2F;hax0rd.txt',       # HTML entities.
     '..&sol;hax0rd.txt',        # HTML entities.
+    '\x00hax0rd\x00.txt\x00',   # Null bytes.
```

Comment:
Is there a reason this is in the `CANDIDATE_TRAVERSAL_FILE_NAMES` rather than the `CANDIDATE_INVALID_FILE_NAMES`?